www-src/app/replicator/main.coffee

```diff
@@ -523,26 +520,3 @@ module.exports = class Replicator extends Backbone.Model
     stopRealtime: =>
         @replicationLauncher?.stop()
         delete @replicationLauncher
-
-    # Update cache files with outdated revisions. Called while backup<
-    syncCache:  (callback) ->
```

Comment:
It's not about picture uploading functionality but file caching.

When you download a file on your phone, the binary is then synced from the cozy: when the file is changed on the cozy, the binary is updated on the phone. That this last feature that need syncCache.